src/components/HolidayRecommendationListModal.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import Modal from '@material-ui/core/Modal';
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles(() => ({
```

Comment:
Isn't Material UI [interoperable](https://material-ui.com/guides/interoperability/#styled-components) with Styled Components? It'd be great to have a consistent approach in styling components across the app unless there was a reason it couldn't be achieved as such. 🙏🏻

Reply:
Good point, updated :)